doc/spire_server.md

```diff
@@ -84,6 +85,17 @@ This may be useful for templating configuration files, for example across differ
 | `attestation`               | Whether or not to rate limit node attestation. If true, node attestation is rate limited to one attempt per second per IP address. | true |
 | `signing`                   | Whether or not to rate limit JWT and X509 signing. If true, JWT and X509 signing are rate limited to 500 requests per second per IP address (separately). | true |
 
+| policy_engine               | Description                    | Default        |
+|:----------------------------|--------------------------------|----------------|
+| `file_provider`             | File to retrieve OPA rego policy for authorization. |      |
```

Comment:
I think it's possible that this may also point to an external OPA endpoint, so there's that possibility that instead of a file it would point to a server with some connection configuration.

Reply:
not sure about other alternatives to OPA, i think there is a realm of possibility but not sure what that would be today... Am in favor of adding an OPA specific config in this case.

Reply:
renamed to `opa_file_provider`